### DISCUSSION TITLE
Stuck on "Please select boot device"

### DISCUSSION THREAD

**Discussion by Azzurite** [elementary/triage#555] [Q&A]
No matter what is pressed here, it goes black for a split second and then comes back to this screen

![image](https://github.com/user-attachments/assets/6fdf9c0b-22eb-4d17-bce2-a400741e26ee)

How do I fix this? I don't have access to another PC right now or get a USB stick, as I'm travelling...

Also, there is no dual boot on this laptop, the disk has been wiped and only elementary os is installed. Why are there 3 entries here in the first place?

**Comment by trmy-m** [comment]
Very interesting boot issue. May I ask what kind of disk the OS is installed to? Looks very much like the OS is dependent on a missing external disk that it was installed to.
